dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/impl/ConfigurableImpl.java

```diff
@@ -152,8 +155,15 @@ public Configuration getConfiguration() {
     }
 
     @Override
-    public C property(String name, Object value) {
-        config.setProperty(name, value);
+    public C property(String name, @Sensitive Object value) {
+        // Liberty Change Start - need to convert proxy password to ProtectedString
+        if ("com.ibm.ws.jaxrs.client.proxy.password".equals(name) && value != null &&
```

Comment:
I don't love that I had to hardcode this property, but I had to because I couldn't ever resolve the constant from `com.ibm.ws.jaxrs.2.0.common`.